roles/delegated_restore/tasks/restore.yml

```diff
@@ -0,0 +1,79 @@
+---
+- name: Stop Odoo container
+  ansible.builtin.command: "{{ docker_compose_bin }} rm -fsv odoo"
+  args:
+    chdir: "{{ inv_compose_dir }}"
+  become: true
+  become_user: "{{ inv_app_user }}"
+
+# Expects db container to be up!
+- name: Drop existing db
+  ansible.builtin.command:
+    argv:
+      - "{{ docker_compose_bin }}"
+      - exec
+      - db
+      - dropdb
+      - --if-exists
+      - -U
+      - odoo
+      - "{{ delegated_dbname }}"
+    chdir: "{{ inv_compose_dir }}"
+  become: true
+  become_user: "{{ inv_app_user }}"
+
+- name: Create new db from template
+  ansible.builtin.command:
+    argv:
+      - "{{ docker_compose_bin }}"
+      - exec
+      - db
+      - createdb
+      - -U
+      - odoo
+      - -T
+      - template0
```

Comment:
To double check: I remember we had some issues due to template1 was used instead of template0 when new DB was created. It might have been related with tableau setup.
Also, what about the access to the database with other tools our customer uses (power bi, tableau or whatever). Will they be able to connect to auto restored databases?

Reply:
Good point. I think its not because of issues, its because we set up tableau user on `template1`, so we could reuse that. This should be possible with that approach  as well. Might be good to make it possible to choose template.